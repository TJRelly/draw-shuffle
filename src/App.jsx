import './App.css'

function App() {

  return (
    <>
     <h1>Draw and Shuffle</h1>
    </>
  )
}

export default App
