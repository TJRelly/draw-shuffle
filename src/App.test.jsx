import { render, fireEvent } from "@testing-library/react";
import { it, expect } from "vitest";
import App from "./App";


// smoke test
it("renders without crashing", function() {
  render(<App />);
});

// snapshot test
it("matches snapshot", function() {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});

