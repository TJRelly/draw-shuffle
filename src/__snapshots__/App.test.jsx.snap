// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="card"
      src="https://deckofcardsapi.com/static/img/back.png"
    />
    <button>
      Draw
    </button>
    <button>
      Shuffle
    </button>
  </div>
</DocumentFragment>
`;
